chpl/chpl-resources/src/main/resources/email.properties

```diff
@@ -27,31 +27,31 @@ questionableActivityNoDataEmailBody=<p>No questionable activity was found betwee
 #pending change request email properties
 pendingChangeRequestReportFilename=pending-change-request
 pendingChangeRequestEmailSubject=Pending Change Request Report
-pendingChangeRequestHasDataEmailBody=<p>A summary of pending change requests is attached.</p><p><ul><li>%s pending Change Website Requests were found</li></ul></p>
-pendingChangeRequestNoDataEmailBody=<p>No pending change requests were found.</p>
+pendingChangeRequestHasDataEmailBody=<p>A summary of pending change requests is attached.</p><p><ul><li>%s pending Change Website Requests were found</li></ul></p>Report contains data for the following ONC-ACBs:
```

Comment:
There should be a %s at the end of these that you'll fill in with the ACB name(s) using String.format in the job

Reply:
I was appending list at the end of this. Do you prefer %s way?